Packs/SaviyntEIC/Integrations/SaviyntEICEventCollector/SaviyntEICEventCollector.yml

```diff
@@ -0,0 +1,98 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: SaviyntEICEventCollector
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+  section: Connect
+- display: Analytics Name
+  name: analytics_name
+  required: true
+  type: 16
+  defaultvalue: SIEMAuditLogs
+  options:
+  - SIEMAuditLogs
+  section: Collect
+- defaultvalue: '50000'
```

Comment:
Do we know how many events are expected?

Reply:
This number is based on the design document.
While the expected event volume is unknown, given that a single request can take up to 30 seconds, I estimated that ingesting 50,000 events will take around 2.5 minutes. This should leave a margin to ingest events and avoid the default 5-minute Docker timeout.